Formula/gdk-pixbuf.rb

```diff
@@ -88,7 +86,7 @@ def post_install
   end
 
   def caveats
-    if build.with?("relocations") || HOMEBREW_PREFIX.to_s != "/usr/local"
```

Comment:
This is not true I think: the formula can perfectly handle non-default prefixes. Best to remove the complete caveat block.

Reply:
Thanks, will do.  I was just being conservative as a first pass.